com.github.antimicro.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "com.github.antimicro",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "antimicro",
+    "finish-args": [
+        "--device=all",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-config/antimicro",
```

Comment:
If they hardcode `~/.config/anitmicro` it should be fixed in the app. If they do not just remove this and it will work.